recipe/conda_build_config.yaml

```diff
@@ -117,16 +121,18 @@ libmatio:
   - 1.5
 libnetcdf:
   - 4.4
+libpcap:
+  - 1.8
 libpng:
-  - 1.6
+  - 1.6.32
 librdkafka:
-  - 0.9.5
+  - 0.9.4
```

Comment:
Same story as `glpk`. Both `0.9.4` and `0.9.5` appear to be API/ABI compatible based on the [report]( https://abi-laboratory.pro/tracker/timeline/librdkafka/ ). So picked the low end of what we had in `pin_the_slow_way.py`.